client/my-sites/site-settings/section-import.jsx

```diff
@@ -119,8 +124,7 @@ export default React.createClass( {
 				{ this.getImports( WORDPRESS ).map( ( importerStatus, key ) =>
 					<WordPressImporter { ...{ key, site, importerStatus } } /> ) }
 
-				{ this.getImports( MEDIUM ).map( ( importerStatus, key ) =>
-					<MediumImporter { ...{ key, site, importerStatus } } /> ) }
+				{ mediumWidget }
```

Comment:
I'd probably prefer to have the conditional block here to keep the output-JSX inline with the others

```js
{ config.isEnabled( 'manage/import/medium' ) &&
	this.getImports( MEDIUM ).map( ( importerStatus, key ) =>
		<MediumImporter { ...{ key, site, importerStatus } } /> );
}
```